postgresql_extensions/README.md

```diff
@@ -53,8 +53,9 @@ The following features are available:
 
 | Name           | Description                               | Default? |
 |----------------|-------------------------------------------|----------|
-| `steampipe`    | Enables Steampipe PostgreSQL extensions   | No       |
-| `tensor-chord` | Enables TensorChord PostgreSQL extensions | No       |
+| `portal-corp`  | Enables PortalCorp PostgreSQL extensions  | Yes      |
+| `steampipe`    | Enables Steampipe PostgreSQL extensions   | Yes      |
+| `tensor-chord` | Enables TensorChord PostgreSQL extensions | Yes      |
```

Comment:
This PR looks really good! I would also suggest you share in the description fields or maybe a new column which Postgres extension this enables. E.g. portal-corp is for pgvector, tensor-chord is for pgvecto.rs

Reply:
I'm wondering if a sorted table with the namespaces and extension name would be useful since there are a large number of extensions for steampipe...